src/Client/Web/Bit.Client.Web.BlazorUI/Components/Inputs/BitTextField.razor.cs

```diff
@@ -64,6 +72,8 @@ public bool IsMultiLine
 
         [Parameter] public EventCallback<MouseEventArgs> OnClick { get; set; }
 
+        public string revealPasswordIconName { get; set; }
```

Comment:
please remove this property and apply the corresponding login in the view itself like the following:
```cs
<i class="bit-icon bit-icon--@(ElementType == TextFieldType.Text ? "Hide" : "RedEye")"></i>
```

Reply:
Done